src/sig_stfl/sig_stfl.c

```diff
@@ -683,3 +683,37 @@ OQS_API void OQS_SIG_STFL_SECRET_KEY_free(OQS_SIG_STFL_SECRET_KEY *sk) {
 	}
 	OQS_MEM_secure_free(sk, sizeof(sk));
 }
+
+OQS_API void OQS_SIG_STFL_SECRET_KEY_SET_store_cb(OQS_SIG_STFL_SECRET_KEY *sk, secure_store_sk store_cb, void *context) {
+	if (sk) {
+		if (sk->set_scrt_key_store_cb) {
+			sk->set_scrt_key_store_cb(sk, store_cb, context);
+		}
+	}
+}
+
+/* Convert LMS secret key object to byte string */
```

Comment:
should be `Convert secret key object to byte string`, right? Because this header is share between XMSS and LMS.

Reply:
Yes, thank you.